app/models/user.rb

```diff
@@ -23,6 +23,13 @@ def default_diet
     Diet.find_or_create_by(name: DEFAULT_DIET_NAME)
   end
 
+  def has_ordered_food_today?
```

Comment:
We actually should have added this line years ago :joy: hard to think when you're outta time. GJ :+1: